src/shared/components/palette/case-flag/components/select-flag-type/select-flag-type.component.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { SelectFlagTypeErrorMessage } from '../..';
+import { SelectFlagTypeComponent } from './select-flag-type.component';
```

Comment:
@johnbenjamin-hmcts - please qualify this import location, i.e. `../../enums`.

Reply:
@Daniel-Lam - Thank you for identifying this. I have fixed this now.